pkg/apis/scorecard/v1alpha3/test_types.go

```diff
@@ -14,6 +14,8 @@ const (
 	FailState State = "fail"
 	// ErrorState occurs when a Test encounters a fatal error and the reported points should not be considered.
 	ErrorState State = "error"
+	//WarnState occurs when a Test's ExpectedPoints == 0 && Test is not required for functionality
```

Comment:
What dictates whether a test is required or optional?

Reply:
Is not the warnState defined by the suggestion : https://github.com/operator-framework/api/blob/d59f62ff6ac73717b98f2ba7be5bb854f80d8910/pkg/apis/scorecard/v1alpha3/test_types.go#L32 ? 

IF we add that what will be the purposed of the suggestion one?